starter_scripts/checksums/Dockerfile.sha512sum

```diff
@@ -1 +1 @@
-5462e09e699649192cc38b2b4e90a561600a3c4f2cc83726d5d8d43cc778aa9f12b549028f782e1b8711100b3d5369a6470e7acedb97db7ecca54d7dc879da4a  Dockerfile
+78cf412f6eaa039681d6ba8bdf25fa10a66ca4157ecdbf5a6090716e9945afa47a44e98e1d3bbbc2c28fef2c7563b59278e63f4470a9ce758c746e511810aeaa  Dockerfile
```

Comment:
do you know, why this changed?

Reply:
because we copy the README into the docker container, see previous file.

Reply:
ah ok, missed that